Essentials/src/com/earth2me/essentials/commands/Commanddelwarp.java

```diff
@@ -16,7 +16,8 @@ public Commanddelwarp() {
     }
 
     @Override
-    public void run(final Server server, final CommandSource sender, final String commandLabel, final String[] args) throws Exception {
+    public void run(final Server server, final CommandSource sender, final String commandLabel, final String[] args)
```

Comment:
Please keep a clean diff and do not change code style or structure in areas of the code that are not immediately relevant to your PR. Code cleanup can be made in a separate PR if you wish.

Reply:
Sorry, auto hard wraps :) Also, the whole class is kinda relevant.